### PR TITLE
Update flood.js

### PR DESCRIPTION
**Pull request by eliah4real**
tell me if it's good

### CODE REVIEW

File: flood.js [hunk 1]
```diff
@@ -26,13 +26,13 @@ console.log("PLEASE GIVE CREDIT TO ME")
...
 //QUESTIONS TO START FLOODING
-pin = readline.question('Enter game pin> ');
-bots = readline.question('Enter number of bots> ');
-ranname = readline.question('Use random name? (y/n)> ');
+pin = readline.question('Enter game pin: ');
+bots = readline.question('Enter number of bots: ');
+ranname = readline.question('Use random name? (y/n): ');
 
-if (ranname=="y"){
+if (ranname == "y" || ranname == "yes"){
```
Comment (eliah4real) [inline flood.js:33]: possibly also add toLowerCase i just don't want to mess it up cause im javascript noob

File: flood.js [hunk 2]
```diff
@@ -26,13 +26,13 @@ console.log("PLEASE GIVE CREDIT TO ME")
...
     ranname=true
-}else{
+} else {
```
Comment (eliah4real) [inline flood.js:35]: cleaner

File: flood.js [hunk 3]
```diff
@@ -41,13 +41,13 @@ er = readline.question('Use name bypass? (y/n)> ');
 
 usercontrolled = readline.question('Controll the bots? (y/n)> ');
 
-if (usercontrolled=="y"){
+if (usercontrolled== "y" || usercontrolled == "yes")){
```
Comment (eliah4real) [inline flood.js:44]: same

File: flood.js [hunk 4]
```diff
@@ -41,13 +41,13 @@ er = readline.question('Use name bypass? (y/n)> ');
...
 
-if (usercontrolled=="y"){
+if (usercontrolled== "y" || usercontrolled == "yes")){
     usercontrolled =true
 }else{
     usercontrolled =false
 }
 
-//END QUESTIONS YAY
+//END QUESTIONS YAYYY
```
Comment (eliah4real) [inline flood.js:50]: more energetic

File: flood.js [hunk 5]
```diff
@@ -169,8 +169,6 @@ function join(name,idee) {
 
                 }
 
-
-
```
Comment (eliah4real) [inline flood.js:173]: clean

File: flood.js [hunk 6]
```diff
@@ -313,4 +309,4 @@ function join(name,idee) {
 console.log("Joining bots")
 const bar = new cliProgress.SingleBar({}, cliProgress.Presets.shades_classic);
 bar.start(bots, 0);
-spam()//YAY WE DID IT
+spam()//YAYYY WE DID IT
```
Comment (eliah4real) [inline flood.js:312]: yay is kinda mehh but YAYYY is good

**Comment by codingMASTER398** [comment]
hey there!

I appreciate your time working on a pull request, and i will add all of these things you suggested, like the ToLowerCase function.

Im on a holiday at the moment, but when i get back i will add all the features in a new release.
This will not be merged through github because im already working on another release, but this will be added in the code, just not merged via github, as i said.

Thanks, you saved me some time.

**Pull request closed without merging**